TestBenches/Macros.h

```diff
@@ -0,0 +1,1083 @@
+// Preprocessor macros used only in C test benches
+#ifndef __SYNTHESIS__
+
+#ifndef TestBenches_Macros_h
+#define TestBenches_Macros_h
+
+// currently defined seeds
+#define __L1L2__ 0
+#define __L2L3__ 1
+#define __L3L4__ 2
+#define __L5L6__ 3
+#define __D1D2__ 4
+#define __D3D4__ 5
+#define __L1D1__ 6
+#define __L2D1__ 7
+
+// processing modules in the current wiring
+#define __VMR_L1PHIA__ 0
+#define __VMR_L1PHIB__ 1
+#define __VMR_L1PHIC__ 2
+#define __VMR_L1PHID__ 3
+#define __VMR_L1PHIE__ 4
+#define __VMR_L1PHIF__ 5
+#define __VMR_L1PHIG__ 6
+#define __VMR_L1PHIH__ 7
+#define __VMR_L2PHIA__ 8
+#define __VMR_L2PHIB__ 9
+#define __VMR_L2PHIC__ 10
+#define __VMR_L2PHID__ 11
+#define __VMR_L3PHIA__ 12
+#define __VMR_L3PHIB__ 13
+#define __VMR_L3PHIC__ 14
+#define __VMR_L3PHID__ 15
+#define __VMR_L4PHIA__ 16
+#define __VMR_L4PHIB__ 17
+#define __VMR_L4PHIC__ 18
+#define __VMR_L4PHID__ 19
+#define __VMR_L5PHIA__ 20
+#define __VMR_L5PHIB__ 21
+#define __VMR_L5PHIC__ 22
+#define __VMR_L5PHID__ 23
+#define __VMR_L6PHIA__ 24
+#define __VMR_L6PHIB__ 25
+#define __VMR_L6PHIC__ 26
+#define __VMR_L6PHID__ 27
+#define __VMR_D1PHIA__ 28
+#define __VMR_D1PHIB__ 29
+#define __VMR_D1PHIC__ 30
+#define __VMR_D1PHID__ 31
+#define __VMR_D2PHIA__ 32
+#define __VMR_D2PHIB__ 33
+#define __VMR_D2PHIC__ 34
+#define __VMR_D2PHID__ 35
+#define __VMR_D3PHIA__ 36
+#define __VMR_D3PHIB__ 37
+#define __VMR_D3PHIC__ 38
+#define __VMR_D3PHID__ 39
+#define __VMR_D4PHIA__ 40
+#define __VMR_D4PHIB__ 41
+#define __VMR_D4PHIC__ 42
+#define __VMR_D4PHID__ 43
+#define __VMR_D5PHIA__ 44
+#define __VMR_D5PHIB__ 45
+#define __VMR_D5PHIC__ 46
+#define __VMR_D5PHID__ 47
+#define __TE_L1PHIA1_L2PHIA1__ 48
```

Comment:
@aehart leading underscores in preprocessor macros are reserved for the compiler by the standard. can you fix this?

https://stackoverflow.com/questions/1449181/what-does-double-underscore-const-mean-in-c/1449301#1449301